### PR TITLE
[CWS] add mode for container running and uuid for fargate

### PR DESCRIPTION
**Pull request by safchain**
### What does this PR do?

Update fargate related metrics.

Add mode on the `datadog.security_agent.runtime.containers_running`
Add uuid on the `datadog.security_agent.runtime.running`

### CODE REVIEW

**Review by paulcacheux — APPROVED** [review]

File: pkg/security/telemetry/telemetry.go [hunk 1]
```diff
@@ -49,7 +50,13 @@ func (c *ContainersTelemetry) ReportContainers(metricName string) {
...
 		}
 
-		c.TelemetrySender.Gauge(metricName, 1.0, []string{"container_id:" + container.ID})
+		var mode string
+		if os.Getenv("ECS_FARGATE") == "true" || os.Getenv("DD_ECS_FARGATE") == "true" {
+			mode = "fargate_ecs"
+		} else if os.Getenv("DD_EKS_FARGATE") == "true" {
+			mode = "fargate_eks"
+		}
+		c.TelemetrySender.Gauge(metricName, 1.0, []string{"container_id:" + container.ID, "mode:" + mode})
```
Comment (paulcacheux) [inline pkg/security/telemetry/telemetry.go:59]: Do we want to include `mode:""` if not on fargate ?

Comment (safchain) [reply pkg/security/telemetry/telemetry.go:59]: going to add default

**Review by paulcacheux — APPROVED** [review]

**Pull request merged**